.env

```diff
@@ -0,0 +1 @@
+NODE_PATH=./
```

Comment:
what this do?

Reply:
this sets up your local node environment to understand where the modules are located based on an absolute path when importing node modules, not technically required for our use case but could be a nice to have